tests/__snapshots__/xlsx_export.test.ts.snap

```diff
@@ -10439,57 +10454,63 @@ Object {
             <diagonal/>
         </border>
     </borders>
-    <cellXfs count=\\"17\\">
+    <cellXfs count=\\"19\\">
         <xf numFmtId=\\"0\\" fillId=\\"0\\" fontId=\\"0\\" borderId=\\"0\\">
-            <alignment vertical=\\"center\\"/>
+            <alignment vertical=\\"bottom\\"/>
         </xf>
         <xf numFmtId=\\"0\\" fillId=\\"0\\" fontId=\\"1\\" borderId=\\"0\\">
-            <alignment vertical=\\"center\\"/>
+            <alignment/>
```

Comment:
Not a real issue, but we shouldn't put empty tags in the xlsx, it's useless

Reply:
@hokolomopo Changes done. I also made `xf` tag self-closed if there's no alignment.